src/jltypes.c

```diff
@@ -2297,7 +2299,7 @@ void jl_init_types(void) JL_GC_DISABLED
                                 jl_any_type,
                                 jl_any_type,
                                 jl_bool_type),
-                        0, 1, 9);
+                        0, 1, 10);
```

Comment:
I guessed I'd need to increment this as this patch needs one more field to be initialized.  Is it the correct usage of `ninitialized` of `jl_new_datatype`?

Reply:
Yes, this is correct because you always initialize `ctxvars` inside `jl_init_root_task` and `jl_new_task`, so it can never be NULL when seen by Julia code.

`ninitialized` is used as a way to keep track of which fields are always initialized, so that, for example, codegen can avoid emitting NULL checks when dereferencing those fields.